File: Greater among three numbers/Greater among three numbers.cpp
#include<iostream>
using namespace std;

int main(){
    int a,b,c;
    cout<<"Enter Three Numbers == > ";
    cin>>a>>b>>c;
    if(a>b && a>c){
        cout<<"First Number is greater";
    }else
    {
        if(b>c){
            cout<<"Second Number is Greater";
        }else{
            cout<<"Third Number is Greater";
        }
    }
    return 0;
}

File: Greater among three numbers/Greater among three numbers.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
Created on Mon Oct  5 11:42:32 2020

@author: ravi
"""
a,b,c = map(int,input("Please Enter three numbers ==> ").split())

if a>b and a>c:
    print("First number is Greatest")
else:
    if b>c:
        print("Second Number is Greatest")
    else:
        print("Third Number is Greatest")